We want to be able to use --with-netlib and --enable-dns at the same time.
RT#2022
--- krb5-1.3.1/src/aclocal.m4	2003-11-24 11:17:30.000000000 -0500
+++ krb5-1.3.1/src/aclocal.m4	2003-11-24 11:18:45.000000000 -0500
@@ -647,6 +647,7 @@
 	LIBS="$LIBS $withval"
 	AC_MSG_RESULT("netlib will use \'$withval\'")
   fi
+  KRB5_AC_ENABLE_DNS
 ],dnl
 [AC_LIBRARY_NET]
 )])dnl
